examples/laptop_pipeline.yml

```diff
@@ -32,6 +32,7 @@ stages:
     - name: TXPSFDiagnostics     # Compute and plots other PSF diagnostics
     - name: TXBrighterFatterPlot # Make plots tracking the brighter-fatter effect
     - name: TXPhotozPlots        # Plot the bin n(z)
+    #- name: TXTwoPGXP            # Plot the gammaX correlation
```

Comment:
Is this supposed to be commented or should we comment it out?

Reply:
Was a left over from trying to get plotting to work

Reply:
Actually @empEvil I just seen that you have some conflicts with master, so you should probably resolve this before merging (sorry I approved too fast! :) )